parasolr/query/queryset.py

```diff
@@ -168,6 +196,17 @@ def get_facets(self) -> Dict[str, Dict]:
         # kwargs results in a Python exception
         return self.solr.query(**query_opts).facet_counts
 
+    def get_stats(self) -> Optional[Dict[str, 'ParasolrDict']]:
+        """Return a dictionary of stats information in Solr format."""
+        if self._result_cache is not None:
+            qr = QueryResponse(self._result_cache)
+            return qr.stats
+        query_opts = self.query_opts()
+        query_opts['rows'] = 0
+        query_opts['hl'] = False
+
+        return self.solr.query(**query_opts).stats
```

Comment:
It probably should. I just rewrote to have it return None in case of a failure, which is a pattern we use elsewhere. The baseline logging in the request loop should mean we have a log of why, and that gives us an easy idiom to check in the Django code (`if stats:`)